chore(website): Update Docs to show information for new Destination Spec option (#9257)
cloudquery/cloudquery: website/pages/docs/reference/source-spec.md

A flag that indicates whether the value of `_cq_id` should be a UUID that is a hash of the primary keys or a random UUID. If a resource has no primary keys defined the value will always be a random UUID

> The latest version of all source plugins support this feature, but only source plugins released after 2023-03-02 will support this functionality. 

Reviewer: Is the date correct?

> The latest version of all source plugins support this feature, but only source plugins released after 2023-03-08 will support this functionality. 

Reviewer 2: ```suggestion
Supported by source plugins released on 2023-03-08 and later
```
